content/content/Usage/plugin management.md

```diff
@@ -0,0 +1,160 @@
+
+Usage:
+  crane plugin-manager add <name> [flags]
+
+Flags:
+  -h, --help             help for add
+      --version string   Install specific plugin version (if not passed, installs latest plugin version or the only available one)
+
+Global Flags:
+  -c, --config-file string   Path to config file which contains a yaml representation of cli flags. Explicit flags take precedence over config file values.
+      --debug                Debug the command by printing more information
+  -p, --plugin-dir string    The path where binary plugins are located (default "plugins")
+      --repo string          The name of the repository from which to list the plugins from
+```
+
+#### Example
+
+```
+crane plugin-manager add foo
+```
+
+The above command downloads the binary of plugin `foo` from appropriate source and places it in plugin directory (default is `plugins`).
+
+### Remove plugin utility
+
+Remove any unwanted plugins that are not needed anymore to exempt them being consumed by transform command.
+
+```
+crane plugin-manager remove -h
+removes the desired plugin
+
+Usage:
+  crane plugin-manager remove <name> [flags]
+
+Flags:
+  -h, --help   help for remove
+
+Global Flags:
+  -c, --config-file string   Path to config file which contains a yaml representation of cli flags. Explicit flags take precedence over config file values.
+      --debug                Debug the command by printing more information
+  -p, --plugin-dir string    The path where binary plugins are located (default "plugins")
+      --repo string          The name of the repository from which to list the plugins from
+```
+
+#### Example
+
+```
+crane plugin-manager remove foo
+```
+
+The above command will remove the plugin `foo` from the plugin directory.
+
+
+### Manual plugin management
+
+Right now there are only two plugins available, there might be more plugins available soon. Currently available plugins are [Kubernetes](https://github.com/konveyor/crane-lib/tree/main/transform/kubernetes) (build into crane-lib) and [OpenShift](https://github.com/konveyor/crane-plugin-openshift). These plugins can be added to desired plugin directory (default is `plugin` directory where `crane` is installed) by following methods -
+
+- Download the binary of the plugin from release and place it in `plugin` directory
+  ```
+  curl -sL https://api.github.com/repos/konveyor/crane-plugin-<plugin-name>/releases/latest | 
+  jq -r ".assets[] | select(.name | contains(\"<arch>-<os>\")) | .browser_download_url" | wget -i-
+  chmod +x <binary>
+  cp <binary> /bin/usr/crane/plugins/<plugin-name>
+  ```
+
+- Build the binary locally and place it in `plugin` directory
+  ```
+  cd $GOPATH
+  git clone https://github.com/konveyor/crane-plugin-<plugin-name>.git
+  cd crane-plugin-<plugin-name>
+  go build -f <plugin-name> .
+  cp <plugin> /bin/usr/crane/plugins/<plugin-name>
+  ```
+
+<b>Note:</b> Adding plugins manually is not advisable as long as it can be added with the help of `plugin-manager`.
```

Comment:
"Adding plugins that are available in the plugin repo is not advisable..."
And maybe a second sentence. "For custom plugins or testing plugins under development, however, manual management is necessary."